template.yaml

```diff
@@ -309,30 +309,30 @@ Resources:
       Timestamp: '${BUILD_TIMESTAMP}'
       AuthorizerArn: !Ref CognitoAuthorizerArn
 
-#  ReadFromScopusBucketsRole:
-#    Type: AWS::IAM::Role
-#    Properties:
-#      AssumeRolePolicyDocument:
-#        Version: '2012-10-17'
-#        Statement:
-#          - Effect: Allow
-#            Principal:
-#              Service: lambda.amazonaws.com
-#            Action: sts:AssumeRole
-#      Path: /
-#      ManagedPolicyArns:
-#        - 'arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole'
-#      Policies:
-#        - PolicyName: ScopusBucketAccess
-#          PolicyDocument:
-#            Version: '2012-10-17'
-#            Statement:
-#              - Effect: Allow
-#                Action:
-#                  - s3:*
-#                Resources:
-#                  - !Sub 'arn:aws:s3:::${ScopusZipBucket}/*'
-#                  - !Sub 'arn:aws:s3:::${ScopusXmlBucket}/*'
+  ReadFromScopusBucketsRole:
```

Comment:
Currently we cannot read from the xml scopus bucket. This is the first step. Probably it would be smarter at a later point to break the roles into policies and compose the roles from sets of policies. Example from publication-api:
```
DefaultLambdaPermissions:
    Type: AWS::IAM::Policy
    Properties:
      PolicyName: !Sub DefaultLambdaPermissions-${AWS::StackName}
      Roles:
        - !Ref LambdaRole
        - !Ref DeleteDraftPublicationLambdaRole
        - !Ref UpdateDoiStatusLambdaRole
      PolicyDocument:
        Version: '2012-10-17'
        Statement:
          - Effect: Allow
            Action:
              - logs:CreateLogGroup
              - logs:CreateLogStream
              - logs:PutLogEvents
              - lambda:InvokeFunction
            Resource: "*"
```